docs/faq.rst

```diff
@@ -123,3 +123,22 @@ Yes, these techniques are complementary.
 Multiband fMRI leads to collecting multiple slices within a volume  simultaneously, while multi-echo
 fMRI is instead related to collecting multiple unique volumes.
 These techniques can be combined to reduce the TR in a multi-echo sequence.
+
+********************************************************************************
+[ME-fMRI] How do field maps and distortion correction work with multi-echo fMRI?
+********************************************************************************
+
+There are many different approaches to susceptibility distortion correction out there- far too many to discuss here.
+However, the good news is that distortion does not typically change across echoes in multi-echo fMRI.
+In most cases, the ``EffectiveEchoSpacing`` is the same across echoes,
+so distortion will remain relatively constant, even as dropout increases with echo time.
+
+What this means is that, in the vast majority of multi-echo datasets,
+standard distortion correction methods will work, and should be applied in the same manner on all echoes.
+For example, if you acquire a blip-up/blip-down set of images for all of your echo times,
+you should use the first echo time's images to generate the undistortion transform,
+as long as that first echo has sufficient gray/white constrast to be useful for alignment
+(in which case, use the earliest echo that does have good contrast).
+
```

Comment:
```suggestion
One potential issue is that the nonlinear spatial transforms that are part of distortion correction methods have the potential to skew the relationship between echoes. Specific examples of nonlinear alignments causing problems have not yet been observed.
```

Reply:
Does this apply when the same transform is applied to all echoes? It seems like this is the problem we're circumventing via our recommendation.

Reply:
I saw this issue on a dataset a while back that had non-trivial intensity inhomogeneity from the surface to the center of the brain. Even though interpolation is calculated using the same voxels in all echoes, the interpolation affected each echo a bit differently and messed up the relationship between the echoes. I really don't know how systematic or serious this issue is, but I've generally been cautious on using aggressive non-linear interpolations before combining echoes.

Reply:
I feel like this has to be rare, though that observation does worry me. The main issue I see with stating something like it is preferable to avoid non-linear alignment is that this is in direct contrast with the default settings in AFNI, for example. 

That said, I think the suggestion is good, because it will keep people on their toes. It does make me think it might be an issue that needs to be raised with AFNI in regards to what data goes into tedana (though its possible I missed some of the flexibility there).